theforeman.org/pipelines/vars/foreman/nightly.groovy

```diff
@@ -7,8 +7,7 @@ def foreman_client_distros = [
     'el7'
 ]
 def foreman_el_releases = [
-    'el9',
-    'el8'
```

Comment:
The pipelines, they are still running

Reply:
Sure, but this PR doesn't disable them ;)

Reply:
Ah, you mean that they should also be dropped from the `centos.org` directory?

Reply:
At least from `def pipelines_el ` in this file :)